apps/dashboard/src/components/billing/highlights-row.tsx

```diff
@@ -49,11 +35,12 @@ function PlanHighlights({ planHighlights }: { planHighlights: Highlight[] }) {
   );
 }
 
-export function HighlightsRow() {
+export function HighlightsRow({ highlightsArray }: HighlightsRowParams) {
+  const numberOfPlans = Object.keys(highlightsArray).length;
   return (
-    <div className="grid grid-cols-1 gap-6 md:grid-cols-3">
-      {Object.entries(highlights).map(([planName, planHighlights]) => (
-        <PlanHighlights key={planName} planHighlights={planHighlights} />
+    <div className={`grid grid-cols-${numberOfPlans} gap-6 md:grid-cols-${numberOfPlans}`}>
```

Comment:
I'm not sure why, but before, it was 1 column on small devices, but now it's not

Reply:
if not needed you can remove `md:grid-cols-...`